packages/stylelint-config-terra/CHANGELOG.md

```diff
@@ -0,0 +1,127 @@
+# ChangeLog
+
+## Unreleased
+
+* Breaking
+  * Drop support for node versions below version 10.
+  * Require stylelint version 13.
+  * Scope package under @cerner.
+
+* Changed
+  * Use jest for tests.
```

Comment:
* Bumped styelint peer dependency to v13

Reply:
thats in the breaking section